test/deploy/k8s/cortx-deployment_backup.yml

```diff
@@ -0,0 +1,57 @@
+apiVersion: apps/v1
```

Comment:
Keep both the files as cortx-deployment.yml  and cortx-statefulset.yml

Reply:
done